draft-ietf-quic-manageability.md

```diff
@@ -809,20 +816,83 @@ off the connection ID may therefore cause undetectable and unrecoverable loss
 of state in the middle of a connection. Use of connection ID specifically
 discouraged for NAT applications.
 
-## Passive Network Performance Measurement and Troubleshooting
+If a NAT hits an operational limit, it is recommended to rather drop the
+initial packets of a flow (see also {{sec-filtering}}),
+which potentially triggers a fallback to TCP, than using the connection ID to
+muliplex mulitple connections on the same IP address port pair which risks
+breakage later on connection ID change.
+
+## Address Rewriting to Ensure Routing Stability
+
+If the routers or switches in the server infrastructure rely on address-port
+4-tuple, a NAT rebinding or any other address migration on the client side,
+even if that could be handled by the QUIC server, will cause packets to be
+delivered to the wrong server. In this case it seems tempting to use NAT in
+front of the server infrastructure to provide a stable address 4-tuple,
+rather than changing the deployed routing infrastructure. As the connection
+ID can change and not-yet-used values are not observable on path, this would
+require servers cooperate to inform the NAT of all new connection IDs,
+similarly as described in {{QUIC_LB}} for load balancers. However, this
```

Comment:
This is not how QUIC-LB works; the servers do not inform the LB about new connection IDs. In fact, there is no way in QUIC-LB to match two connection IDs to the same connection, merely to the same server.